osquery/tables/other/augeas.cpp

```diff
@@ -0,0 +1,50 @@
+#include <osquery/tables.h>
+
+#include <augeas.h>
+#include <boost/algorithm/string/join.hpp>
+
+namespace osquery {
+namespace tables {
+void match(QueryData &results, const augeas *aug, const char *pattern) {
```

Comment:
+ newline about this

Also, please `osquery::tables::match` is far too generic of a symbol name. Please change it to matchAugeasThingy or something.